src/modelldcatnotordf/modelldcatno.py

```diff
@@ -25,6 +26,8 @@
 FOAF = Namespace("http://xmlns.com/foaf/0.1/")
 SKOS = Namespace("http://www.w3.org/2004/02/skos/core#")
 XKOS = Namespace("http://rdf-vocabulary.ddialliance.org/xkos#")
+OWL = Namespace("http://www.w3.org/2002/07/owl#")
+ADMS = Namespace("http://www.w3.org/ns/adms#")
```

Comment:
Sjekk hvilke namespaces går an å importere fra rdflib i stedet for lage her.

Se denne:
https://github.com/RDFLib/rdflib/blob/8e3c771a5c6a2787b5a34a9288c7d5d6967ecbbe/rdflib/namespace.py#L48

Reply:
Ah, ok, takk!